mifospay/src/main/java/org/mifos/mobilewallet/mifospay/home/BaseHomeContract.java

```diff
@@ -17,8 +17,6 @@
 public interface BaseHomeContract {
 
     interface BaseHomeView extends BaseView<BaseHomePresenter> {
-
-        void showClientDetails(Client client);
```

Comment:
Since we don't have any methods inside the interface, we can skip it altogether. If any requirement arises for the same in the future, we can always create it again